### PR TITLE
Removed obsolete snippets

### PR DESCRIPTION
**Pull request by pkulikov**
Removed snippets that are not used as the result of dotnet/docs#12411 and dotnet/docs#12426

### CODE REVIEW

File: snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers.cs [hunk 1]
```diff
@@ -1,131 +1,3 @@
-//using System;
-//using System.Runtime.InteropServices;
-//using System.Text;
-
-
-//<Snippet2>
```
Comment (pkulikov) [inline snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers.cs]: Not related to the mentioned PRs, but I also didn't find usings of this snippet.

Comment (rpetrusha) [reply snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers.cs]: @pkulikov, I'm finding that this file is still in use:

- docs\csharp\programming-guide\unsafe-code-pointers\pointer-conversions.md: [!code-csharp[csProgGuidePointers#4](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers.cs#4)]
- docs\csharp\programming-guide\xmldoc\processing-the-xml-file.md:[!code-csharp[csProgGuidePointers#21](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers.cs#21)]

File: snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers2.cs [hunk 2]
```diff
@@ -1,41 +1,3 @@
-//<Snippet1>
```
Comment (rpetrusha) [inline snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers2.cs]: This also still seems to be in use:

programming-guide\unsafe-code-pointers\pointer-conversions.md: [!code-csharp[csProgGuidePointers#3](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers2.cs#3)]

File: snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers.cs (hunk 1 above)
Comment (pkulikov) [reply snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers.cs]: @rpetrusha yes, I've found those two usages. That's why the whole file *is not being removed*. Only  not used parts of it.

File: snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers2.cs (hunk 2 above)
Comment (pkulikov) [reply snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers2.cs]: Only `<Snippet3>` is referenced.

File: snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers.cs (hunk 1 above)
Comment (rpetrusha) [reply snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers.cs]: Sorry, @pkulikov -- I should have noted that the entire file wasn't deleted!

File: snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers2.cs (hunk 2 above)
Comment (rpetrusha) [reply snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers2.cs]: Again, I should have noticed that, @pkulikov. I apologize.

**Comment by rpetrusha** [comment]
Thanks for removing unneeded files and unused code snippets, @pkulikov. I'll merge your PR now.

**Pull request merged**